readthedocs/api/v2/views/footer_views.py

```diff
@@ -243,16 +242,6 @@ def get(self, request, format=None):
             'version_supported': version.supported,
         }
 
-        # Allow folks to hook onto the footer response for various information
-        # collection, or to modify the resp_data.
-        footer_response.send(
-            sender=None,
-            request=request,
-            context=context,
-            response_data=resp_data,
-            absolute_uri=self.request.GET.get('absolute_uri'),
-        )
-
```

Comment:
Shouldn't we keep sending this signal here?

We were only using it for pageviews here, but it could be useful for other things as well. Besides, it may have an impact downstream.

Reply:
Don't think this is useful, it also hides additional queries. The class can be extended already with `OverrideSettginsObject`, that's better than using signals for this case.

Reply:
My only worry is about custom installs that might be using it, but I don't expect it to be heavily used there. _We_ plan to cache it in prod, but not all users will necessarily do that. I think I'm OK removing it, and we can re-add it if someone yells :)